polyfill/test/datetime.mjs

```diff
@@ -299,7 +299,13 @@ describe('DateTime', () => {
     });
     it('year ≤ 99', () => {
       const dt = DateTime.from('+000098-10-29T10:46:38.271986102');
-      equal(`${dt.inTimeZone('+06:00')}`, '+000098-10-29T04:46:38.271986102Z');
+      equal(`${dt.inTimeZone('+06:00')}`, '+000098-10-29T16:46:38.271986102Z');
```

Comment:
Does this change belong in the previous commit?

Reply:
It actually didn't belong here at all, it was a rebase error. (I'm a bit surprised the test didn't fail.)